salt/utils/pkg/rpm.py

```diff
@@ -59,16 +60,17 @@ def check_32(arch, osarch=None):
     return all(x in ARCHES_32 for x in (osarch, arch))
 
 
-def pkginfo(name, version, arch, repoid):
+def pkginfo(name, version, arch, repoid, install_date, install_date_time_t):
```

Comment:
Yes, I did take a look at the call tree and it only boils down to `yumpkg` basically in the places interested by this PR. So it should be safe FMPOV.

Thanks!